pages/workspace/workspace.html.js

```diff
@@ -206,11 +206,19 @@ function Workspace({ navTitle, showLogs, resLocals }) {
         <main class="d-flex flex-column flex-grow h-100">
           <div
             id="loading"
-            class="d-flex flex-grow h-100 justify-content-center align-items-center"
+            class="d-flex h-100 flex-grow justify-content-center align-items-center"
           >
             <i class="d-block fa fa-10x fa-circle-notch fa-spin text-info" aria-hidden="true"></i>
             <span class="sr-only">Loading workspace &hellip;</span>
           </div>
+          <div
+            id="stopped"
+            class="d-none h-100 flex-grow flex-column justify-content-center align-items-center"
+          >
+            <h2>Workspace stopped due to inactivity</h2>
+            <p>Your data was automatically saved. Reload the page to restart the workspace.</p>
```

Comment:
@mwest1066 I know we had talked about communicating the specific time limits involved, but I wasn't sure how to communicate the intricacies of "your tab was in the background for too long" vs. "your client didn't heartbeat for a while" and the fact that each of those use different timeouts. Hopefully this generic message is sufficient?

Reply:
I think this is fine. No need to get more specific.